tests/src/Integration/Condition/ConditionManagerTest.php

```diff
@@ -11,6 +11,8 @@
 
 /**
  * Tests the Rules condition manager.
+ * @group rules_conditions
```

Comment:
Overall I think this pull request is a good idea.

The initiative to make rules - "testbot friendly" is a good idea.
 
This is a minor nit pick there should be a blank line between the text and the group tag in the code fragment above.

It will show up on a phpcs error report.